### PR TITLE
feat(frontend): add info box in activity page

### PR DESCRIPTION
**Pull request by AntonioVentilii**
# Motivation

We include an info box in the activity page to warn about third-parties on BTC transactions.

My understanding of the requirements is that we always show it, independently of having or not BTC transactions, hence there is no logic to show it or not.

Screenshot 2024-11-14 at 11 00 17
Screenshot 2024-11-14 at 11 00 43
Screenshot 2024-11-14 at 11 00 49

### CODE REVIEW

**Review by peterpeterparker — COMMENTED** [review]
I did not had a look at the code yet.

- Also no logic to close the information with the flag saved in local storage similar to what we already do?

- It's expected that the style is different that the existing? The existing will be modified?

See screenshot.
Capture d’écran 2024-11-14 à 11 35 40

**Comment by AntonioVentilii** [comment]
> Also no logic to close the information with the flag saved in local storage similar to what we already do?

this one will be provided in another PR, but not for now

> It's expected that the style is different that the existing? The existing will be modified?


Correct, this is the new info box component from the Figma library. Not really sure if the others will be modified too, to be honest, but I can ask.

**Comment by peterpeterparker** [comment]
> but I can ask.

If you don't mind please do ask if the existing box design should be updated. Consistency is my passion 😉.

Thanks!

File: src/frontend/src/lib/components/ui/MessageBox.svelte [hunk 1]
```diff
@@ -1,18 +1,19 @@
 <script lang="ts">
 	import IconInfo from '$lib/components/icons/lucide/IconInfo.svelte';
 
-	export let level: 'info' | 'light-warning' | 'error' = 'info';
+	export let level: 'plain' | 'info' | 'light-warning' | 'error' = 'info';
```
[the remark this answers is not in the document]
Comment (AntonioVentilii) [reply src/frontend/src/lib/components/ui/MessageBox.svelte:4]: on Figma, they have several levels for this message box, and they asked to be with white background (defined as `plain`) to have more contrast. The level `info` has a bluish background

[the remark this answers is not in the document]
Comment (peterpeterparker) [reply src/frontend/src/lib/components/ui/MessageBox.svelte:4]: What I mean is "level === plain", what's a level "plain"?

But all good if it's use in Figma makes sense to use it here as well.

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

**Comment by AntonioVentilii** [comment]
@peterpeterparker I asked @artkorotkikh-dfinity to have a look: he will probably give a direct feedback or maybe ask @StefanBerger-DFINITY to follow up on this

**Comment by artkorotkikh-dfinity** [comment]
I'm late! But: 
- It's expected to have a new style - we'll use these info boxes in the future as well
- The idea was actually to have it closable. I think we can apply the same logic as in the existing Info boxes! 

image

**Comment by AntonioVentilii** [comment]
[OISY-130]

**Pull request merged**